exchanges/coinbasepro/coinbasepro_test.go

```diff
@@ -76,7 +76,7 @@ func TestGetTrades(t *testing.T) {
 }
 
 func TestGetHistoricRates(t *testing.T) {
-	_, err := c.GetHistoricRates(testPair, 0, 0, 0)
+	_, err := c.GetHistoricRates(testPair, "0", "0", 0)
```

Comment:
This test is doesn't really cover this now that its implemented.
Please update with real values and verify that data is returned. eg
```
func TestGetHistoricRates(t *testing.T) {
	iso8601format := "2006-01-02T15:04:05"
	end := time.Now().UTC()
	start := time.Now().UTC().Add(-time.Second * 300)
	resp, err := c.GetHistoricRates(testPair, start.Format(iso8601format), end.Format(iso8601format), 0)
	if err != nil {
		t.Error(err)
	}
	if len(resp) == 0 {
		t.Error("Expected results")
	}
}
```